tests/rkt_rm_test.go

```diff
@@ -99,3 +99,22 @@ func TestRmInvalid(t *testing.T) {
 	cmd = fmt.Sprintf("%s rm --uuid-file=%s", ctx.Cmd(), uuidFile)
 	runRktAndCheckOutput(t, cmd, expected, true)
 }
+
+func TestRmEmptyUUID(t *testing.T) {
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	emptyUUID := "\"\""
+	nonexistentUUID := "0f746094-3438-42bc-ab37-3cf85f132e60"
+	tmpDir := createTempDirOrPanic("rkt_rm_test")
+	defer os.RemoveAll(tmpDir)
+	uuidFile := filepath.Join(tmpDir, "uuid-file")
+	if err := ioutil.WriteFile(uuidFile, []byte(nonexistentUUID), 0600); err != nil {
+		t.Fatalf("cannot write uuid-file: %v", err)
+	}
```

Comment:
Yeah this can be deleted, I was not sure if it was necessary to have a some kind of data to be able to run `rkt rm` correctly, like the example before. I suppose that the creation of the uuid-file it is a precaution to avoid deleting an existing pod, isn't it?.